src/items/rqgItem.ts

```diff
@@ -174,6 +174,11 @@ export class RqgItem extends Item {
     rqid: string,
     lang: string = "en"
   ): Promise<RqgItem | undefined> {
+
+    if (!rqid || rqid === "") {
```

Comment:
The second part is unnecessary since empty string is "falsy"
```suggestion
    if (!rqid) {
```